### PR TITLE
Skin jankenpon

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: cordova/www/games/jankenpon/jankenpon.js [hunk 1]
```diff
@@ -0,0 +1,223 @@
+var Jankenpon = {
+  introDialog: [
+    "\u30ef\u30a6!! IT'S TIME.....to play.....",
+    "JAN--KEN--PON!!!!", 
+    "What?! Of course I know Japanese, " + Game.getUsername() + "-san!",
```
Comment (varffs) [inline cordova/www/games/jankenpon/jankenpon.js]: do we need to check that this is a regular naming convention and works for both genders?

File: cordova/www/games/jankenpon/jankenpon.js [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+    
+  },
+
+  startGame: function() {
+    var _this = this;
+
+    _this.bind();
+  },
+
+  logScore: function() {
```
Comment (varffs) [inline cordova/www/games/jankenpon/jankenpon.js]: does this need to exist anymore?

File: cordova/www/games/jankenpon/jankenpon.js [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+
+  },
+
+  hideResult: function() {
+    var _this = this;
+
+    _this.$userChoice.children( 'img' ).css('display', 'none');
+    _this.$masterChoice.children( 'img' ).css('display', 'none');
+    _this.$result.css({
+      'display': 'none',
```
Comment (varffs) [inline cordova/www/games/jankenpon/jankenpon.js]: why the different formatting here?

**Comment by mikeyrayvon** [comment]
k those issues resolved

**Pull request merged**